analog-commander/usr/share/asteroid-launcher/watchfaces/analog-commander.qml

```diff
@@ -0,0 +1,499 @@
+/*
+ * Copyright (C) 2022 - Ivo Hulsman <github.com/ivohulsman>
+ *               2021 - Timo Könnecke <github.com/eLtMosen>
+ *               2016 - Sylvia van Os <iamsylvie@openmailbox.org>
+ *               2015 - Florent Revest <revestflo@gmail.com>
+ *               2012 - Vasiliy Sorokin <sorokin.vasiliy@gmail.com>
+ *                      Aleksey Mikhailichenko <a.v.mich@gmail.com>
+ *                      Arto Jalkanen <ajalkane@gmail.com>
+ * All rights reserved.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation, either version 2.1 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import QtQuick 2.1
+import QtGraphicalEffects 1.15
+import Nemo.Mce 1.0
+import org.asteroid.controls 1.0
+import org.asteroid.utils 1.0
+
+Item {
+    id: root
+    property string imgPath: "../watchfaces-img/analog-commander-"
+    property real rad: 0.01745
+    MceBatteryLevel {
+        id: batteryChargePercentage
+    }
+    Repeater {
```

Comment:
This formatting of this `Repeater` is OK, but the others are a bit inconsistent.  I'd recommend formatting them all the same way and also recommend being consistent about whether `model` or `id` should be first.